tests/helper/helper_kubectl.go

```diff
@@ -203,3 +210,10 @@ func (kubectl KubectlRunner) DeletePod(podName string, namespace string) {
 func (kubectl KubectlRunner) WaitAndCheckForTerminatingState(resourceType, namespace string, timeoutMinutes int) bool {
 	return WaitAndCheckForTerminatingState(kubectl.path, resourceType, namespace, timeoutMinutes)
 }
+
+func (kubectl KubectlRunner) VerifyResourceDeleted(resourceType, resourceName, namespace string) {
```

Comment:
I would say let's call `VerifyResourceDeleted` common function for `oc` and `kubectl` into the interface here https://github.com/openshift/odo/blob/main/tests/helper/helper_cli.go

Reply:
Ahh my bad. I saw its already there 🙂